pyodk/_endpoints/submissions.py

```diff
@@ -213,14 +220,26 @@ def create(
           </meta>
           <name>Alice</name>
           <age>36</age>
+          <selfie>profile.jpg</selfie>
         </data>
         ```
 
+        If provided, each attachment file name must match a file name reference (just the
+        file name, not a directory path) in the submission XML, otherwise Central will
+        reject the upload since the file is not expected. For example, an `image` question
+        named `bird` with a XML value `<bird>pigeon.jpg</bird>` will allow uploading an
+        attachment named `pigeon.jpg`. Central assumes that clients (you), will ensure
+        unique file name references in the submission XML if necessary, e.g. Central will
+        only store one `pigeon.jpg` file across all questions in a given submission. You
+        may provide directory paths to pyodk e.g. `["/home/u/pigeon.jpg"]`, but only the
+        file name is used to identify the file in the upload to Central.
```

Comment:
This is helpful, I agree. What do you think about also mentioning that chunk size can be set on the session? Users might not realize that's available/helpful in this context without a nudge.